.gitignore

```diff
@@ -1,4 +1,3 @@
-config.yml
```

Comment:
This change can be backed out. We don't want to check-in a config.yml for the reason you state.